homeassistant/components/rflink/__init__.py

```diff
@@ -372,6 +372,11 @@ def _handle_event(self, event):
         """Platform specific event handler."""
         raise NotImplementedError()
 
+    @property
+    def unique_id(self):
+        """Return the unique ID of this entity."""
+        return self._device_id
```

Comment:
This looks a bit strange. Cant there be multiple sensors per device?

Reply:
No.
TL;DR;
The RFLink library split every device report (with multiple sensors measurements) in diferents sensor reports (one for measurement).
That `_device_id` was an unlucky name for the attribute.
In previous refactorings I tried to change all references to `entity_id` myself, but the PR was ultimately not accepted.
So, that `device_id` refers to the entity id and not the device (that yes, can have multiple sensors)

Reply:
This should be set in init, and use the self._attr_unique_id